source/win-main/App.vue

```diff
@@ -766,6 +804,9 @@ export default defineComponent({
             this.$closePopover()
           }
         })
+    },
+    getToolbarButtonDisplay: function (configName: string): boolean {
+      return this.$store.state.config['displayToolbarButtons.' + configName] === true
```

Comment:
Just for confirmation: You checked that, when someone changes one of the settings in the preferences the toolbar automatically updates, correct?

Reply:
> 



> Just for confirmation: You checked that, when someone changes one of the settings in the preferences the toolbar automatically updates, correct?

Yes, when the user changes the setting of toolbar preferences, the toolbar automatically updates.

Reply:
Perfect, thank you for confirming!